embedo.js

```diff
@@ -399,7 +406,7 @@
       options = options || {};
       var target = document.head || document.getElementsByTagName('head')[0];
       var script = document.createElement('script');
-      var jsonpCallback = 'embedo_fetch_' + Embedo.utils.uuid();
+      var jsonpCallback = 'embedofetch' + Embedo.utils.uuid();
```

Comment:
Callback request sample: ~~~&callback=embedo_fetch_1nhwnq
However, spotify tends to search for embedofetch1nhwnq().